CHANGES.md

```diff
@@ -3,10 +3,18 @@ Change Log
 
 ### 1.9 - 2015-05-01
 * Breaking changes
-  *
+  * Removed `ColorMaterialProperty.fromColor`. Pass a `Color` directly to the `ColorMaterialProperty` constructor instead.
```

Comment:
We usually say what version something was deprecated in, e.g.,
> Removed `Camera.setTransform`, which was deprecated in Cesium 1.6. Use `Camera.lookAtTransform`.

I don't know that it is strictly necessary but it does give the impression that we are not just making breaking changes, especially for new users that might not know our deprecation policy.